raster-foundry/raster-foundry
# Use backsplash for multiband mosaics

## Overview

This PR uses `backsplash` for multiband mosaics.

### Checklist


### Notes

Still todo:

- [x] (halfway) global histograms with histogram cacheing so colors aren't terrible
- [x] fetch avro scenes also instead of having a nice not-implemented in there

## Testing Instructions

 * `./scripts/server --quiet backsplash`
 * go directly to a project (the projects splash page is pretty destructive to the tile server, for some reason) -- with avro scenes only since cog histogram calculation is currently brutal in the sense that it blows the java heap :scream:
 * load some tiles
 * look, they're tiles
 * make some similar requests with a bogus token and observe that you get rejected
 * make requests for a different project that you don't have access to with a good token and observe that you get rejected
 * change the band combinations
 * grep for `Future` and feel smug

Closes #3786 

Closes #3903
